src/Sarif.ValidationTests/DefaultObjectPopulatingVisitor.cs

```diff
@@ -109,9 +109,10 @@ public override Node VisitNode(Node node)
             return node;
         }
 
-        private void PopulateInstanceWithDefaultMemberValues(ISarifNode node)        {
-           
+        private void PopulateInstanceWithDefaultMemberValues(ISarifNode node)
```

Comment:
>PopulateInstanceWithDefaultMemberValues [](start = 21, length = 39)

FYI, these tests were very helpful in understanding our serialization/deserialization behaviors. They operate by producing data dynamically, by comprehensively walking/reflecting over the OM and poking values to various things. This is my nirvana for unit-testing: no inlined code, no data files driving testing (with unknown comprehensiveness).